ampeg.testing/src/test/java/HomePage.java

```diff
@@ -0,0 +1,70 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class HomePage extends PageObject {
+
+	public HomePage(WebDriver driver) {
+		super(driver);
+	}
+	
+	public SearchResultsPage enterSearch() {
+		var searchIcon = this.driver.findElement(By.cssSelector("a[title=Search]"));
+		searchIcon.click();
+		var searchBar = this.driver.findElement(By.cssSelector("#searchq"));
+		searchBar.sendKeys("svt-810e");
+		searchBar.submit();
+		
+		return new SearchResultsPage(this.driver);
+	}
+	
+	public HistoryPage clickHistoryLink() {
+		var menuLink = this.driver.findElement(By.cssSelector("a[title=History]"));
+		menuLink.click();
+		
+		return new HistoryPage(this.driver);
+	}
+	
+	public SupportPage clickSupportLink() {
+		var supportLink = this.driver.findElement(By.cssSelector("a[title=Support]"));
+		supportLink.click();
+		
+		return new SupportPage(this.driver);
+	}
+	
+	public ArtistPage clickArtistLink() {
+		var artistLink = this.driver.findElement(By.cssSelector("a[title=Artists]"));
+		artistLink.click();
+		
+		return new ArtistPage(this.driver);
+	}
+	
+	public HomePage signIn() {
+		var loginIcon = this.driver.findElement(By.cssSelector("a[title=Account]"));
+		loginIcon.click();
+		var userNameBox = this.driver.findElement(By.cssSelector("input[name=l_user]"));
+		userNameBox.sendKeys("chaseglasspoole");
```

Comment:
move the username and password to method parameters so you can use this method to sign in as any user